src/core/ext/xds/certificate_provider_store.h

```diff
@@ -83,9 +84,17 @@ class CertificateProviderStore
       return certificate_provider_->interested_parties();
     }
 
+    int cmp_impl(const grpc_tls_certificate_provider* other) const override {
```

Comment:
In this case, I think it should delegate to the `Compare()` method of the wrapped provider.

Reply:
Modified the TODO. (I would really prefer to mess with these in this PR.)